### DISCUSSION TITLE
Signin button has a rounded-md tw style

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#1551] [Anything Else]
"Thought you should know the signin button has a rounded-md tw style with the wrapping div having rounded, causing it to look a bit strange in the corners, see attached photo."

<img width="471" alt="Skärmavbild 2021-12-18 kl  19 22 17" src="https://user-images.githubusercontent.com/34340819/146668464-e470b577-1fde-4cd6-b48a-204e420bcb17.png">

**Comment by ukutaht** [comment]
Fixed [here](https://github.com/plausible/website/commit/b65f55a34e93189e35082e18381cd1646d169a98)
